### ISSUE TITLE
Don't duplicate class attribute

### ISSUE THREAD

**Issue by os0x** [k0kubun/hamlit#4]
```haml
.item{ class: 'first' }
```

## Expected
```html
<div class='item first'></div>
```

## Actual
```html
<div class='item' class='first'></div>
```

**Comment by k0kubun** [comment]
Thank you for reporting the issue. :pray: :bow:

**Comment by k0kubun** [comment]
Fixed and released as v0.5.1.
Thank you for your testing :)
